ocfweb/docs/docs/staff/procedures/new-host.md

```diff
@@ -138,3 +138,13 @@ until the configuration converges.
 The first Puppet run and various other things may be broken if one or more
 packages are out of date, e.g. Puppet. Remedy this with an `apt-get update &&
 apt-get upgrade`.
+
+## Tips and Tricks
+
+### VNC monitoring of Debian install
```

Comment:
Might be useful to mention `virt-manager` here, but it does require the root password to use, so it's not the easiest, probably better to encourage `virt-viewer` instead.